frontend/src/components/panels/NodeGraph.svelte

```diff
@@ -46,6 +46,19 @@
 	$: nodeCategories = buildNodeCategories($nodeGraph.nodeTypes, searchTerm);
 	$: nodeListX = ((nodeListLocation?.x || 0) * GRID_SIZE + transform.x) * transform.scale;
 	$: nodeListY = ((nodeListLocation?.y || 0) * GRID_SIZE + transform.y) * transform.scale;
+
+	let appearAboveMouse = false;
+	let appearRightOfMouse = false;
+
+	$: (() => {
+		const bounds = graph?.div()?.getBoundingClientRect();
+		if (!bounds) return;
+		const { width, height } = bounds;
+
+		appearRightOfMouse = nodeListX > width - 90;
+		appearAboveMouse = nodeListY > height - 100;
```

Comment:
Perhaps we should declare the constants `90` and `100` at the top of the `<script>` tag for easy access?

Reply:
yes, that does make it clearer.